demisto_sdk/commands/common/hook_validations/content_entity_validator.py

```diff
@@ -101,7 +103,10 @@ def is_release_branch():
         Returns:
             (bool): is release branch
         """
-        diff_string_config_yml = run_command("git diff origin/master .circleci/config.yml")
+        git_util = GitUtil(repo=Content.git())
+        main_branch = git_util.handle_prev_ver()[1]
```

Comment:
should this be a function in `tools.py` or `git_util.py`?

Reply:
It's relevant when git is used, so I say keep it in git_util.py

Reply:
ok, then let's move it to `git_util.py`

Reply:
Do you mean like a static method?